Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WatchTab/config.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import formatMessage from 'format-message';
+
+import { TabExtensionConfig, WatchTabKey } from '../types';
+
+import { WatchTabHeader } from './WatchTabHeader';
+import { WatchTabContent } from './WatchTabContent';
+
+export const WatchTabConfig: TabExtensionConfig = {
+  key: WatchTabKey,
+  description: () => formatMessage('Watch tab'),
```

Comment:
this wont work when locale changes, convert the whole thing to a function

Reply:
I'm not familiar with how the strings for each locale are generated. What needs to be converted to a function here?